dispatchrouter/pom.xml

```diff
@@ -94,7 +94,7 @@
               <images>
                 <image>
                   <build>
-                    <from>eclipsehono/qpid-dispatch:${qpid-dispatch.version}</from>
+                    <from>enmasseproject/qdrouterd:0.8.0</from>
```

Comment:
Is this an *official* image that is being built as part of enmasse continuously? We could consider to use this image then instead of rolling our own, assuming that it contains all sasl mechanism plugins we consider relevant.
Do you have any insight?